src/components/button-toggle/button-toggle.style.ts

```diff
@@ -65,6 +67,40 @@ const StyledButtonToggle = styled.button<StyledButtonToggleProps>`
   box-sizing: border-box;
   max-width: 100%;
 
+  ${({ variant }) =>
+    variant === "minor" &&
+    css`
+      border: 1px solid var(--colorsActionMinor500);
+
+      &&&&&& {
+        border-radius: var(--borderRadius050);
+        padding: var(--spacing000) var(--spacing100) var(--spacing000)
+          var(--spacing100);
+      }
+
+      &:focus {
+        border: 4px solid var(--colorsActionMinor500);
+        background-color: var(--colorsActionMinor600);
+      }
+
+      &&&:hover {
```

Comment:
comment: see above for suggested change, I don't think we should use `&&&` but instead use the same style attribute to target it that we are trying to override etc